2022q3/cloud-init.adoc

```diff
@@ -8,14 +8,14 @@ link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATI
 Contact: Mina Galić <me+FreeBSD@igalic.co>
 
 cloud-init is the standard way of provisioning servers in the cloud.
-Unfortunately, cloud-init’s support for operating systems other than Linux is rather poor and the lack of cloud-init support on FreeBSD is a hindrance to cloud providers who want to offer FreeBSD as a Tier 1 platform.
-To remedy this, this project aims to bring FreeBSD cloud-init support on par with Linux support. Though the plan is to lift support across all BSDs.
+Unfortunately, cloud-init support for operating systems other than Linux is rather poor, and the lack of cloud-init support on FreeBSD is a hindrance to cloud providers who want to offer FreeBSD as a Tier 1 platform.
+To remedy the situation, this project aims to bring FreeBSD cloud-init support on par with Linux support. The broader plan is to lift support across all BSDs.
```

Comment:
Nit (AsciiDoc): I should have started a new line for the second of these two sentences.